docs/CHANGELOG.rst

```diff
@@ -25,6 +25,8 @@ Changed
   ``resolwebio/base:ubuntu-18.04`` Docker image as a base image in
   ``resolwebio/bamliquidator:1.1.0`` Docker image
 - Use ``resolwebio/bamliquidator:1.2.0`` in ``rose2`` process
+- Fix missing entity type in ``seqtk-rev-complement-single``,
```

Comment:
This PR includes a fix for the not-yet-released feature, so it would be better to skip the changelog entry. However, leave it as it is, since we will merge the related bullet points when preparing the final release notes.